src/components/molecules/MarketStats.tsx

```diff
@@ -19,9 +22,29 @@ interface MarketStatsResponse {
 
 const refreshInterval = 60000 // 60 sec.
 
+const getTotalPoolsValues = gql`
+  query PoolsData {
+    poolFactories {
+      totalLockedValue
```

Comment:
just a heads up, over in `ocean-subgraph` multiple fixes were added today, among them fixes for TVL value but also fixing the naming of the keys to be correct and consistent. So `totalLockedValue` is now correctly `totalValueLocked`: https://github.com/oceanprotocol/ocean-subgraph/blob/main/src/mappings/factory.ts#L15

So once that's deployed (always a bit unclear if `main` is deployed or not), we need to switch key in the query here and also make sure that within this PR we always use `totalValueLocked` e.g for variable names